hw/top_earlgrey/ip/pwrmgr/data/autogen/pwrmgr.hjson

```diff
@@ -346,7 +346,10 @@
             { value: "1",
               name: "Enabled",
               desc: '''
-                USB clock enabled during low power state
+                USB clock enabled during low power state.
+
+                However, if !!CONTROL.MAIN_PD_N is 0, USB clock is disabled
+                during low power state.
```

Comment:
It seems this corresponding sentence should be added to the enabled setting of the other clocks as well.

Reply:
i just realized i missed this comment. Will update.